src/main/java/seedu/address/logic/commands/ConfirmPrompt.java

```diff
@@ -0,0 +1,39 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+
+/**
+ * Prompts the user to confirm the operation.
+ */
+public class ConfirmPrompt extends Command {
+    public static final String MESSAGE_CONFIRM_PROMPT = "Please type in command 'confirm' to confirm the operation.\n"
+            + "Otherwise, to cancel the operation.";
+
```

Comment:
Perhaps you should be more clear about what is otherwise, y/n might be better? or what should the user input to tell them to cancel operation (maybe a cancel command class is needed?)